### PR TITLE
Rails 5 generator

### PR DESCRIPTION
**Pull request by RyanBibby**
The sessions table migration that is generated currently always inherits from `ActiveRecord::Migration`. This is no longer supported in Rails 5.1. This pull request will make it inherit from `ActiveRecord::Migration[MAJOR.MINOR]` for Rails 5+ applications.

### CODE REVIEW

File: lib/generators/active_record/templates/migration.rb [hunk 1]
```diff
@@ -9,4 +9,4 @@ def change
     add_index :<%= session_table_name %>, :session_id, :unique => true
     add_index :<%= session_table_name %>, :updated_at
   end
-end
+end
```
Comment (robin850) [inline lib/generators/active_record/templates/migration.rb]: Could leave the very last blank line in the file please ? This is intentional.

File: lib/generators/active_record/session_migration_generator.rb [hunk 2]
```diff
@@ -19,6 +19,10 @@ def session_table_name
           end
           current_table_name
         end
+
+        def migration_version
+          "[#{ActiveRecord::VERSION::MAJOR}.#{ActiveRecord::VERSION::MINOR}]" if ActiveRecord::VERSION::MAJOR >= 5
```
Comment (robin850) [inline lib/generators/active_record/session_migration_generator.rb]: There's a short-hand for this:

~~~ruby
def migration_version
  "[#{ActiveRecord::Migration.current_version}]" if ActiveRecord::Migration.respond_to?(:current_version)
end
~~~

**Comment by RyanBibby** [comment]
Thank you very much. I have made those changes now.

**Comment by RyanBibby** [comment]
The Travis failure looks to be happening due to a failure while bundling ruby 1.9.3. As far as I can tell, all that has changed since the last success is the rubygems version. 

I don't think this is something I can fix on my end, but if it is please let me know.

**Comment by RyanBibby** [comment]
Thanks very much. I've mad that change now.

**Comment by jamgregory** [comment]
@robin850 - do you know when this will be available as an update to the gem please? :+1:

**Pull request merged**